src/trio/_core/_run.py

```diff
@@ -540,11 +540,21 @@ class CancelScope:
     _has_been_entered: bool = attrs.field(default=False, init=False)
     _registered_deadline: float = attrs.field(default=inf, init=False)
     _cancel_called: bool = attrs.field(default=False, init=False)
+    _is_relative: bool | None = attrs.field(default=False, init=False)
     cancelled_caught: bool = attrs.field(default=False, init=False)
 
     # Constructor arguments:
-    _deadline: float = attrs.field(default=inf, kw_only=True, alias="deadline")
-    _shield: bool = attrs.field(default=False, kw_only=True, alias="shield")
+    _relative_deadline: float = attrs.field(default=inf, kw_only=True)
+    _deadline: float = attrs.field(default=inf, kw_only=True)
+    _shield: bool = attrs.field(default=False, kw_only=True)
```

Comment:
I think we want to keep the non-underscored `alias=` for each of these.

Reply:
The non-underscored explicit alias doesn't do anything, attrs automatically exposes them as non-underscored init parameters unless you specify `init=False`.
We could keep them for the sake of being extra explicit, but attrs docs don't do it and it could lead to confusion in the case of somebody not having an alias.